scio-avro/src/test/scala/com/spotify/scio/avro/types/TypeProviderTest.scala

```diff
@@ -21,45 +21,39 @@ import com.google.protobuf.ByteString
 import com.spotify.scio.avro.types.AvroType.HasAvroDoc
 import org.apache.avro.Schema
 import org.apache.avro.generic.GenericRecord
-import org.scalatest.{FlatSpec, Matchers}
+import org.scalatest.{Assertion, FlatSpec, Matchers}
+
+import scala.annotation.StaticAnnotation
+import scala.reflect.runtime.universe._
 
 class TypeProviderTest extends FlatSpec with Matchers {
   @AvroType.fromSchema(
     """{"type":"record","name": "Record","fields":[{"name":"f1","type":"int"}]}""")
   class StringLiteralRecord
 
   @AvroType.fromSchema(
-    """
-      {"type":"record","name": "Record","fields":[{"name":"f1","type":"int"}]}
-    """)
+    """{"type":"record","name": "Record","fields":[{"name":"f1","type":"int"}]}""")
   class MultiLineStringLiteral
 
   @AvroType.fromSchema(
-    """
-      |{"type":"record","name": "Record","fields":[{"name":"f1","type":"int"}]}
-    """.stripMargin)
+    """{"type":"record","name": "Record","fields":[{"name":"f1","type":"int"}]}""")
```

Comment:
what's the reason to remove the `stripMargin`

Reply:
The length of the file went beyond the permissible limits (800 lines per file). I had to go for such tricks. If this is critical, I can return. What do you think?

Reply:
@nevillelyh what do you think

that seems like a silly reason to remove stylistic elements and things that we're testing

Reply:
You can also surround this with a pair of `// scalastyle:{off,on} line.size.limit` to suppress the limit.